Maestro/subscriptions.json

```diff
@@ -78,9 +78,9 @@
             "-Project='tests\\build.proj'",
             "--",
             "/t:UpdateDependenciesAndSubmitPullRequest",
-            "/p:GitHubUser=dotnet-bot",
-            "/p:GitHubEmail=dotnet-bot@microsoft.com",
-            "/p:GitHubAuthToken=`$(`$Secrets['DotNetBotGitHubPassword'])",
+            "/p:GitHubUser=dotnet-maestro-bot",
+            "/p:GitHubEmail=dotnet-maestro-bot@microsoft.com",
+            "/p:GitHubAuthToken=`$(`$Secrets['DotNetMaestroBotGitHubToken'])",
```

Comment:
I took a look at Maestro-CoreFXGeneralExecutor, and it looks like you added the variable, but didn't hook it up to the powershell script args:

    @{"DotNetBotGitHubPassword"="$(DotNetBotGitHubPassword)"}

Should be:

    @{"DotNetBotGitHubPassword"="$(DotNetBotGitHubPassword)";"DotNetBotMaestroGitHubPassword"="$(DotNetBotMaestroGitHubPassword)"}